source/open-definition-2.1-dev.markdown

```diff
@@ -42,8 +42,7 @@ accompany the **work**.
 
 The **work** *must* be provided in an open format. An open format is
 one which places no restrictions, monetary or otherwise, upon its use and can be fully processed
-with at least one free/libre/open-source software tool. Data *must* be machine-readable and 
-*should* be provided in bulk.
+with at least one free/libre/open-source software tool. The **work** *should* be provided in the form preferred for making modifications to it.
```

Comment:
I don't understand why work should be bold in this second use. Otherwise looks good to me